code/lib/create-storybook/src/initiate.ts

```diff
@@ -337,7 +337,11 @@ export async function doInitiate(options: CommandOptions): Promise<
     selectedFeatures = new Set(out.features);
   }
 
-  const telemetryFeatures = ['dev', ...selectedFeatures];
+  const telemetryFeatures = {
+    docs: selectedFeatures.has('docs'),
+    test: selectedFeatures.has('test'),
+    dev: true,
```

Comment:
```suggestion
    dev: true,
    docs: selectedFeatures.has('docs'),
    test: selectedFeatures.has('test'),
```

Just to keep it alphabetical.